test/parse.js

```diff
@@ -19,7 +19,12 @@ var Path = require('path');
 // http://mxr.mozilla.org/mozilla-central/source/js/src/tests/ecma_5/JSON/
 
 var dirsPath = Path.resolve(__dirname, 'parse-cases');
-var dirs = FS.readdirSync(dirsPath);
+var dirs = FS.readdirSync(dirsPath).filter(function(fn) {
+  if (fn.startsWith('.')) {
```

Comment:
@ianloic Thanks for the fix. Making changes to your tree is perfect since I can merge those in and you still get credit for the commits.